BrainPortal/lib/portal_system_checks.rb

```diff
@@ -266,5 +266,19 @@ def self.z020_start_background_activity_workers #:nodoc:
 
   end
 
+  def self.z030_ensure_to_load_oidc_config
+
+      #----------------------------------------------------------------------------
+      puts "C> Loading OIDC configuration..."
+      #----------------------------------------------------------------------------
+
+      begin
+        OidcConfig.load_from_file
+      rescue => ex
+        puts "C> \t- ERROR: Cannot load OIDC configuration."
```

Comment:
Put a better message here, say something like `check the content of "oidc.yml.rb"`

Reply:
Also, make it stop the server boot process.